Makefile

```diff
@@ -42,9 +42,9 @@ JSON_FILES	 = \
 #
 
 NAME 			= minnow
-NODE_PREBUILT_TAG	= zone
+NODE_PREBUILT_TAG	= zone64
```

Comment:
Will the jump to 64-bit cause any problems?  Does "make test" work?

Reply:
It should be fine. This module is almost pure JS. All it does is check the free disk space and post it to moray, so it's pretty simple. And unfortunately, there's no test suite for it.

The reason I made this change is because I had an issue creating a jenkins-agent-1.6.3 instance that I could use for build testing so I decided to try a later image.

Minnow itself doesn't get an image, it's installed into the storage (mako) image which is on 19.4.0. But minnow uses node v0.10, and I didn't want to bother with dealing potential breaking node API changes. 15.4.1 is the last version we have v0.10 builds of sdcnode for so I picked 15.4.1 for the base image.

When I did the push, the jenkins job got stuck because we don't have any i386 15.4.1 jenkins agents, so it got bumped from zone to zone64.

But I don't believe any of this will cause an issue.